test/tests/functional/pbs_multi_sched.py

```diff
@@ -2170,7 +2170,7 @@ def test_standing_confimred_resv_reconfirm(self):
         """
         self.common_setup()
         now = int(time.time())
-        self.degraded_resv_reconfirm(start=now + 20, end=now + 200,
```

Comment:
Since the test does not wait for the reservation to start, why not use now + 600 and now + 800 just to be safe?